backend/tests/core/DatabaseHelper.cs

```diff
@@ -61,10 +88,23 @@ public static PimsContext CreatePimsContext(this TestHelper helper, string dbNam
             var context = new PimsContext(options, contextAccessor.Object);
 
             if (ensureDeleted) context.Database.EnsureDeleted();
+            helper.AddSingleton(context);
 
             return context;
         }
 
+        /// <summary>
+        /// Initialize the PimsContext with the specified data in 'entities'.
+        /// Initializes the database with default data to support other tables.
+        /// </summary>
+        /// <param name="helper"></param>
+        /// <param name="user"></param>
+        /// <returns></returns>
+        public static PimsContext InitializeDatabase(this TestHelper helper, ClaimsPrincipal user)
+        {
+            return helper.InitializeDatabase(StringHelper.Generate(10), user);
```

Comment:
By default it will generate an unique PimsContext and add it to the service provider.  This reduces the amount of boilerplate code in tests.